daemon/graphdriver/btrfs/btrfs.go

```diff
@@ -274,6 +407,16 @@ func (d *Driver) Create(id, parent, mountLabel string, storageOpt map[string]str
 		}
 	}
 
+	if len(storageOpt) != 0 {
```

Comment:
It might be better to be a little more explicit here.
AFAICT if another storage opt is supported  later on that has nothing to do with the storage size we'll still set one anyway (or at least try to) and potentially get an error.